BL-402 Updated product doc repository

EstrellaDionis opened the pull request:
## Description

Update product doc repository to include searching case data

#### Video Link

https://www.loom.com/share/7d67a629306b4da5932f2b7d2e733281

#### Trello Link

https://bloomtechlabs.atlassian.net/jira/software/c/projects/BL/boards/8?modal=detail&selectedIssue=BL-402

(Example below):

<blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;JfipwNkb&#x2F;265-fix-refactor-sidebar-to-have-icons-representing-each-element-in-the-sidebar">Update product doc repository to include searching case data</a></blockquote>

## Checklist:

- [x] My design/diagram follows the style guidelines of this project
- [x ] My docs are organized within an appropriate folder structure in our repo
- [x ] Pull request description clearly describes docs made & motivations for said additions

EstrellaDionis committed:
updated description in pull_request.md

Jason7jl reviewed (commented):
Great job!

vbrambila2 committed:
BL-549 added style guide template

vbrambila2 committed:
BL-549 updated Link to ticket

MaxwellChrist committed:
BL-408 created component layouts according to style guid as seen in figma, added mock heatmap image to one of the four components

MaxwellChrist commented on pull_request_template.md:
```diff
@@ -1,6 +1,6 @@
 
-___Add your PR description here, deleting this example___
+Update product doc repository to include searching case data
 
```
Why are we changing this? Isn't the line you're trying to replace in the template specific to what you did for this pull request? I get it if this was part of the searchandfilter.md file but not in the generic template

vbrambila2 replied:
I believe that was mistake, I'll reach out to Dionis and get it figured out. Good catch though, we don't want to mess with the original template file.

MaxwellChrist reviewed (commented):
I agree that adding the search and filter information inside the design folder is a good idea and I see that change in the repo already. Looking at the files changed for this request however, I'm a little confused that the only change I see is on the pull request template

attaris978 committed:
BL-570 feature/Update API docs

Updated API docs to reflect stakeholder-requested ability to filter using multiple selections from several search categories. Removed deprecated schema/query parameters (race/ethnicity, caseId). Added explanatory comments and notes on implementation.

VWashingtonCoder reviewed (approved):
Great job guys. That was definitely a good way to condense the design folder. I believe this should make it easier for future devs to not only read the specific designs they need but also find them in the first place!

attaris978 reviewed (changes requested):
Could you change the word "Trello" to "Jira" in the pull request doc, please?

billydean commented:
To @attaris978 's point, we should probably just change the "Trello" reference in the PR template, to save us the trouble. If y'all are making this change anyway, maybe you could go ahead and change it there, too.

vbrambila2 committed:
BL-402 changed Trello to Jira in template

attaris978 committed:
Updated formatting

Per feedback from Eric Lambert and Will Kim, updated to match openapi 3.0.0 specifications. Boom. 😮😮😮

Ricardorouge reviewed (approved):
I am a little newer to this project so I'm not really sure of the changes made but from what I can see it looks very organized and easy to maneuver good job

Joshmclemore reviewed (approved):
Love it. This is much more organized

vbrambila2 committed:
BL-402 changed template to Trello to Jira

vbrambila2 committed:
BL-402 updated template

vbrambila2 commented:
> Could you change the word "Trello" to "Jira" in the pull request doc, please?

Update is complete, Trello has been changed to Jira.

MaxwellChrist reviewed (approved):
I really like the organization you all did for this ticket. Also I really appreciate the explanation at the end of the loom video. The comments left in the yaml file were a nice addition. Overall nothing but great changes made so keep up the good work

EricMarshallLambert reviewed (changes requested):
Good work!
I like that we can easily watch the videos and link to the resources so we can edit them. However, I think it would be helpful to also include an image file(s) of the resource within the folder so the overall documentation can link to them directly.

1. Please update thepull_request_template.md from Trello to Jira.

Mbravo36 reviewed (approved):
Making and organizing those folders is very helpful to the project, looks good.

shannonsullivan reviewed (changes requested):
Nice organization of the Designs folder. A couple of comments/questions:
- EngineeringArchitecture - Can we remove this folder completely? This design is outdated. There is an updated version in  the asylum-rg-be-a backend repo.
- StyleGuide & StyledComponents - Can we combine these folders as the both references the same figma link? One is an updated version of the other.
- UIMockup - Is there more information that we can include in this folder? An image of the file or is there another version of an .UiMockup.pdf.icloud file that can be opened easier?
- As mentioned above, would it be possible to save an image from the Figma file, put it in a resources folder in the repo and link the image to the pull request template markdown file along with the Figma link?

EstrellaDionis committed:
Removed EngineeringArchitecture folder

EstrellaDionis committed:
Created Styles folder to house all styles

EstrellaDionis commented:
> Nice organization of the Designs folder. A couple of comments/questions:
> 
> * EngineeringArchitecture - Can we remove this folder completely? This design is outdated. There is an updated version in  the asylum-rg-be-a backend repo.
> * StyleGuide & StyledComponents - Can we combine these folders as the both references the same figma link? One is an updated version of the other.
> * UIMockup - Is there more information that we can include in this folder? An image of the file or is there another version of an .UiMockup.pdf.icloud file that can be opened easier?
> * As mentioned above, would it be possible to save an image from the Figma file, put it in a resources folder in the repo and link the image to the pull request template markdown file along with the Figma link?

EstrellaDionis committed:
updated trello's to jira where applicable

EstrellaDionis committed:
changed trello's to jira's where applicable

shannonsullivan reviewed (approved):
Changes look good!

XavierTwitty reviewed (approved):
Good job updating the file, everything looks very organized.
